### ISSUE TITLE
No YouTube Icon

### ISSUE THREAD

**Issue by martincostello** [justeat/f-footer#23]
Our UK consumer website currently has a YouTube link in its footer.

We should have an SVG, to match the other social icons, for YouTube.

**Comment by DamianMullins** [comment]
Is there a Jira to track this?

**Comment by martincostello** [comment]
No, I haven't made one as I assumed this was enough. Also I'd be wary of putting links to our internal JIRA here in a public repo.

**Comment by DamianMullins** [comment]
We do need to figure out the process for this. 

Also what concerns do have regarding putting Jira's on this repo? I'm pretty sure we've added them elsewhere already...
